crt_portal/cts_forms/views.py

```diff
@@ -536,18 +536,19 @@ def get(self, request):
         return render(request, 'forms/complaint_view/actions/index.html', output)
 
     def post(self, request):
-        bulk_actions_form = BulkActions(request.POST)
         return_url_args = request.POST.get('next', '')
         selected_all = request.POST.get('all', '') == 'all'
         confirm_all = request.POST.get('confirm_all', '') == 'confirm_all'
         ids = request.POST.get('ids', '').split(',')
 
-        if bulk_actions_form.is_valid():
-            if confirm_all:
-                requested_query = self.reconstruct_query(return_url_args)
-            else:
-                requested_query = Report.objects.filter(pk__in=ids)
+        if confirm_all:
+            requested_query = self.reconstruct_query(return_url_args)
+        else:
+            requested_query = Report.objects.filter(pk__in=ids)
 
+        bulk_actions_form = BulkActions(request.POST, requested_query)
```

Comment:
Not related to the functionality here but I'd love to rename `BulkActions` to  `BulkActionsForm` and help nudge us towards more idiomatic Django class names.